openstack/client.go

```diff
@@ -197,6 +197,19 @@ func NewIdentityV3(client *gophercloud.ProviderClient) *gophercloud.ServiceClien
 	}
 }
 
+func NewIdentityAdmin(client *gophercloud.ProviderClient) (*gophercloud.ServiceClient, error) {
+	eo := gophercloud.EndpointOpts{
```

Comment:
Yes, but my question is: If the endpoint can change by region, and I want the endpoint for a particular region, how would I pass in the `region` to specify which endpoint I wanted? Other (non-identity) services pass a `gophercloud.EndpointOpts` to the function, which contains the region.